dt-core/setup-functions.php

```diff
@@ -70,3 +70,31 @@ function dt_setup_roles_and_permissions(){
         }
     }
 }
+
+/**
+ * Add custom roles and permissions from the roles manager.
+ */
+add_filter( 'dt_set_roles_and_permissions', 'dt_setup_custom_roles_and_permissions', 11, 1 );
+
+function dt_setup_custom_roles_and_permissions( $roles ) {
+    $custom_roles = get_option( 'dt_custom_roles' );
```

Comment:
@incraigulous i get an invalid arg error if no dt_custom_roles is set.
Suggestion:
`$custom_roles = get_option( 'dt_custom_roles', [] );`